### PR TITLE
Update the tutorial docs for 'assets'

### PR DESCRIPTION
**Pull request by ganicke**
Add/expand some sections to the tutorial docs for Asset support.

RE: #7899

### CODE REVIEW

**Review by Jaqster — APPROVED** [review]
Looks good to me!

**Comment by ganicke** [comment]
Maybe the "Resources" page should be the "Assets" page?

File: docs/writing-docs/tutorials/resources.md [hunk 1]
```diff
@@ -79,11 +79,69 @@ controller.A.onEvent(ControllerButtonEvent.Pressed, function () {
...
+for Arcade's beta editor would be:
 
 https://arcade.makecode.com/beta?savetemplate=1
 
-With the correct URL entered, you can click the save button (the floppy disk) in the editor to download your project as a `.txt.mkcd` file. The contents of this file can be opened in a text editor and pasted into the ```` ```assetsjson ``` ```` block to configure the tutorial. To reopen and edit the project, simply drag the original `.txt.mkcd` into the editor and repeat the process to download it again.
\ No newline at end of file
+With the correct URL entered, you can click the save button (the floppy disk button) in the
+editor to download your project as a `.txt.mkcd` file.
+
+Next, define a ```` ```assetsjson ```` block in your tutorial document.
```
Comment (shakao) [inline docs/writing-docs/tutorials/resources.md:107]: ah! this should be singular `assetjson`, the previous docs were wrong

Comment (ganicke) [reply docs/writing-docs/tutorials/resources.md:107]: Eeek! Ok.

**Review by shakao — APPROVED** [review]

File: docs/writing-docs/tutorials/resources.md (hunk 1 above)
Comment (shakao) [reply docs/writing-docs/tutorials/resources.md]: thanks galen!!

**Pull request merged**